src/pages/CartPage/components/CartItemList.tsx

```
@@ -0,0 +1,66 @@
+            <span className="w-8 text-center">{item.quantity}</span>
+            <button
+              onClick={() => onUpdateQuantity(item.id, item.quantity + 1)}
+              className="flex h-8 w-8 items-center justify-center rounded-full border border-slate-300 hover:bg-slate-50"
+            >
+              +
+            </button>
+          </div>
+          <button
+            onClick={() => onRemoveItem(item.id)}
+            className="text-sm text-red-600 hover:text-red-700"
+          >
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)